ArmPose init_arm_pose
geometry_msgs/Pose wrist_target
geometry_msgs/Pose eff_target
---
ArmPose arm_pose